Documentation/ExtensionArchitecture/ComposerJson/Index.rst

```diff
@@ -0,0 +1,85 @@
+.. include:: ../../Includes.txt
+
+
+.. _composer-json:
+
+=====================
+composer.json
+=====================
+
+*-- required*
+
+.. note::
+
+   While the file :file:`composer.json` is currently not strictly required, it is considered
+   bad practice not to add one. That is why we classify it as "required".
+
+Including a :file:`composers.json` is strongly recommended for a number of reasons:
+
+#. The file :file:`composer.json` is required for documentation rendering since
+   May 29, 2019.
+
+   See :ref:`h2document:migrate` for more information on the necessary changes for
+   extension documentation rendering.
+
+#. Working with Composer in general is strongly recommended for TYPO3.
+
+   If you are not using Composer for your projects yet, see :ref:`t3install:migrate-to-composer`
+   in the "Installation & Upgrade Guide".
+
+
+Minimal composer.json
+=====================
+
+This is a minimal composer.json for a TYPO3 extensions:
```

Comment:
Depending on what amount of extensions we're talking about, this needs to be structured a bit differently.

Either it is 

> This is a minimal composer.json for a TYPO3 extension:

or

> This is a minimal composer.json for TYPO3 extensions: